backend/package.json

```diff
@@ -17,14 +17,18 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "typeorm": "typeorm"
```

Comment:
you need to add migration command here according to typeORM documentation. Then give migration files too with this PR so we will run migrations directly

"migration:generate": "npm run typeorm -- migration:generate",

Reply:
done